rest/config.go

```diff
@@ -767,9 +767,16 @@ func RunServer(config *ServerConfig) {
 
 	sc := NewServerContext(config)
 	for _, dbConfig := range config.Databases {
-		if _, err := sc.AddDatabaseFromConfig(dbConfig); err != nil {
+		c, err := sc.AddDatabaseFromConfig(dbConfig)
+		if err != nil {
 			base.LogFatal("Error opening database: %v", err)
 		}
+
+		seq, err := c.LastSequence()
+		if err != nil {
+			base.LogFatal("Error getting last sequence for %s: %v", dbConfig.Name, err)
+		}
+		base.LogTo("Cache", "_sync:seq for %s: %d", dbConfig.Name, seq)
```

Comment:
Wasn't 100% certain which log key I should be using for this. May have to be changed.

Reply:
For one-time startup logging like this I think we should just use base.Logf().